modules/distr-tracing-tempo-install-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-operator_{context}"]
+= Installing the {TempoName} Operator
+
+To install {TempoName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {TempoName} Operator.
+
+By default, the Operator is installed in the `openshift-operators` project.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {TempoName} Operator.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Type *distributed tracing platform* into the filter to locate the {TempoName} Operator.
+
+. Click the *{TempoName} Operator* provided by Red Hat to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Install Operator* page, select the *stable* Update Channel. This automatically updates your Operator as new versions are released.
+//If you select a maintenance channel, for example, *Stable*, you will receive bug fixes and security patches for the length of the support cycle for that version.
+
+. Accept the default *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
```

Comment:
@andreasgerstmayr I think the namespace here should be `openshift-tempo-operator` right?

Reply:
This is already fixed.